L1Trigger/TrackFindingTracklet/interface/DiskResidual.h

```diff
@@ -0,0 +1,134 @@
+//Base class for processing modules
+#ifndef DISKRESIDUAL_H
+#define DISKRESIDUAL_H
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
(will not comment on all, just mark as resolved, all "using namespace std" in a header file are fixed in the updated version.